frontend/cypress/integration/common/sidecar_injection.ts

```diff
@@ -0,0 +1,76 @@
+import { Given, Then, When } from "cypress-cucumber-preprocessor/steps";
+
+Given('a namespace without override configuration for automatic sidecar injection', function () {
```

Comment:
Minor, we are setting the namespaces somehow explicitly.

"A namespace..." could mean anything, and I'd need to look into the implementation to know which namespace it refers.

It could be nice to have "the default namespace...", that could help to read better these tests.

Reply:
> "A namespace..." could mean anything

Indeed, and I wrote it that way on purpose. The namespace name is not relevant as long as it fulfills the statements. These `Given` functions are making sure that such statements become fulfilled. So, it is quite possible to switch to another namespace in the implementation and the tests will still pass.